NewHorizons/Assets/translations/russian.json

```diff
@@ -11,6 +11,7 @@
     "FREQ_WARP_CORE": "Гиперядро",
     "FREQ_UNKNOWN": "???",
     "ENGAGE_WARP_PROMPT": "Телепортироваться к {0}",
-    "WARP_LOCKED": "АВТОПИЛОТ ЗАКРЕПЛЁН НА:\n{0}"
+    "WARP_LOCKED": "АВТОПИЛОТ ЗАКРЕПЛЁН НА:\n{0}",
+    "Vessel": "«Звездолёта»"
```

Comment:
The << and >> are quotation marks in Russian

Reply:
There is none. I looked at the base game translations for dialogue talking about vessel and just took it from there.

Reply:
respect